### PR TITLE
Update README.md.

### PR DESCRIPTION
**Pull request by renormalize**
**What this PR does / why we need it**:

* Fix multiple typos.

* Format the document to adhere to Markdown conventions.

**Release note**:

```doc operator
Update README.md
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 
 [![REUSE status](https://api.reuse.software/badge/github.com/gardener/ops-toolbelt)](https://api.reuse.software/info/github.com/gardener/ops-toolbelt)
 
-### What is the ops-toolbelt
-The `ops-toolbelt` aims to be a standard operator container image with pre-installed useful tools for troubleshooting issues on gardener landscapes. The `ops-toolbelt` images can be used by the Gardener Dashboard web consoles functionality to log into the garden, seed, or shoot clusters.
+## What is the ops-toolbelt?
 
-The pods created with this image can be both general pods and node-bound pods (behaving as if being on the node directly)
-Starting a pod with the ops-toolbel timage requires a running Kubelet and healthy control plane, a working VPN connection, and sufficient capacity on the node.
+The `ops-toolbelt` aims to be a standard container image with pre-installed useful tools for troubleshooting issues on gardener landscapes for human operators. The `ops-toolbelt` images can be used by the Gardener Dashboard web consoles functionality to log into the garden, seed, or shoot clusters.
```
Comment (petersutter) [inline README.md:7]: > The `ops-toolbelt` images can be used by the Gardener Dashboard web consoles functionality to log into the garden, seed, or shoot clusters.

not sure we want to mention it here

File: README.md [hunk 2]
```diff
@@ -58,17 +63,20 @@ $
 
 Use `./hacks/ops-pod --help` to check what other options are available
 
+## Building ops-toolbelt images
 
-### Building ops-toolbelt images
 Dockerfiles for the images are generated from files in the `dockerfile-config` directory.
 
 To build all pre-configured images run:
+
```
Comment (petersutter) [inline README.md:71]: Please update the sentence above 'To build all pre-configured images run:', as it is no longer accurate. Previously, we built a matrix of images with different cloud provider CLIs, but we no longer do this. Now, we only build one image.

**Review by petersutter — APPROVED** [review]
thank you for updating the docs! some minor feedback, otherwise /lgtm

File: README.md (hunk 1 above)
Comment (renormalize) [reply README.md:7]: Yeah, I wasn't sure either but I didn't want to remove things which I wasn't sure about! Thanks for the feedback.

File: README.md (hunk 2 above)
Comment (renormalize) [reply README.md:71]: Ah right. Thanks for the suggestion!

File: README.md (hunk 1 above)
Comment (renormalize) [reply README.md]: Addressed in [0102cad](https://github.com/gardener/ops-toolbelt/pull/142/commits/0102cad035b1842c8cb187e8f670ff6aa5a7cf45).

File: README.md (hunk 2 above)
Comment (renormalize) [reply README.md:71]: Addressed in [0102cad](https://github.com/gardener/ops-toolbelt/pull/142/commits/0102cad035b1842c8cb187e8f670ff6aa5a7cf45).

**Comment by renormalize** [comment]
@petersutter let me know if you have any more suggestions. Thanks.

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**